index.js

```diff
@@ -1,23 +1,22 @@
-if (typeof window !== 'undefined') {
-  const navMenuIcon = document.querySelector('.nav-menu-icon');
-  const navLinks = document.querySelector('.nav-links');
+const navMenuIcon = document.querySelector('.nav-menu-icon');
 
-  navMenuIcon.addEventListener('click', () => {
-    navMenuIcon.classList.toggle('active');
-    navLinks.classList.toggle('active');
-  });
+const navLinks = document.querySelector('.nav-links');
+
+navMenuIcon.addEventListener('click', () => {
+  navMenuIcon.classList.toggle('active');
+  navLinks.classList.toggle('active');
+});
 
-  const closeIcon = document.querySelector('.close-icon');
-  closeIcon.addEventListener('click', () => {
+const closeIcon = document.querySelector('.close-icon');
+closeIcon.addEventListener('click', () => {
+  navMenuIcon.classList.toggle('active');
+  navLinks.classList.toggle('active');
+});
+
+const menuItem = document.querySelectorAll('.menu-item');
+menuItem.forEach((menu) => {
+  menu.addEventListener('click', () => {
     navMenuIcon.classList.toggle('active');
     navLinks.classList.toggle('active');
   });
-
-  const menuItem = document.querySelectorAll('.menu-item');
-  menuItem.forEach((menu) => {
-    menu.addEventListener('click', () => {
-      navMenuIcon.classList.toggle('active');
-      navLinks.classList.toggle('active');
-    });
-  });
-}
\ No newline at end of file
+});
```

Comment:
- We are not working with JavaScript yet, please delete this file until the next activities demand it. 👊🏽